tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -0,0 +1,162 @@
+```
+
+</div>
+
+<div label="MyCLI Client">
+
+[MyCLI](https://www.mycli.net/) enables TLS automatically with the `ssl` flag. You can set `ssl-ca` to enable TLS and validate TiDB serverless Tier clusters.
+
+- Use `--ssl-ca=<CA_root_path>` to set the CA root path in your system.
+
+```shell
+mycli -u <username> -h <host> -P 4000 -D test --ssl-ca=<CA_root_path>
+```
+
+</div>
+
+<div label="Java">
+
+We recommend using [MySQL Connector/J 8.0](https://dev.mysql.com/doc/connector-j/8.0/en/) or later versions to connect to TiDB Serverless Tier clusters in Java.
+
+- Set `sslMode` in connection string to enable TLS and validate TiDB Serverless Tier clusters. JDBC trusts system CA root certificates by default, so you do not need to configure certificates.
+- With `--ssl-mode=VERIFY_IDENTITY`, JDBC forces to enable TLS and validate TiDB Serverless Tier clusters.
+
+```
+jdbc:mysql://<host>:4000/test?user=<username>&password=<your_password>&sslMode=VERIFY_IDENTITY
+```
+
+</div>
+
+<div label="Python">
+
+We recommend using [mysqlclient](https://pypi.org/project/mysqlclient/) to connect to TiDB Serverless Tier clusters in Python. 
+
+- Set `ssl.ca` in the connection string to enable TLS and validate TiDB Serverless Tier clusters.
+- Use `ssl={"ca": "<CA_root_path>"}` to set the CA root path in your system.
+
+```
+host="<host>", user="<username>", password="<your_password>", port=4000, database="test", ssl={"ca": "<CA_root_path>"}
+```
+
+</div>
+
+<div label="Go">
+
+It is recommended to use the [Go-MySQL-Driver](https://github.com/go-sql-driver/mysql) to connect to TiDB Serverless Tier clusters in Go.
+
+- Register `tls.Config` in connection to enable TLS and validate TiDB Serverless Tier clusters. Go uses system CA root certificates by default, so you do not need to configure certificates.
+- Register `tidb` TLS configuration. Set `ServerName: "<host>"` to verify TiDB Serverless Tier's hostname.
+- If you do not want to register a new TLS configuration, you can just set `tls=true` in the connection string.
+
+```
+mysql.RegisterTLSConfig("tidb", &tls.Config{
+  MinVersion: tls.VersionTLS12,
+  ServerName: "<host>",
+})
+
+db, err := sql.Open("mysql", "<usename>:<your_password>@tcp(<host>:4000)/test?tls=tidb")
+```
+
+</div>
+
+<div label="Node.js">
+
+It is recommended to use [mysql2](https://www.npmjs.com/package/mysql2) to connect with TiDB Serverless Tier clusters in Node.js. Node.js use system CA root certificates by default, so you do not need to configure certificates anymore.
+
+- Set `ssl: {"rejectUnauthorized": true}` to validate TiDB Serverless Tier's hostname.
+
+```
+host: '<host>', port: 4000,user: '<username>', password: '<your_password>', database: 'test', ssl: {"rejectUnauthorized": true}
+```
+
+</div>
+</SimpleTab>
+
+## Where is the CA root path in my system?
+
+The following lists the CA root paths in common platforms.
```

Comment:
Do we need to provide the root cert for users who's os do not contains our root cert

Reply:
It's mentioned in the preview sections. You could download the CA cert from Let's Encrypt or CA bundle from Mozilla.